src/screens/WalletDetailsScreen.tsx

```diff
@@ -42,11 +42,15 @@ interface Props {
 
 export class WalletDetailsScreen extends React.PureComponent<Props> {
   componentDidMount() {
-    this.checkSubscription();
+    const { wallet, email, checkSubscription } = this.props;
```

Comment:
please keep the `this.checkSubscription();` this function does exactly what is written now in componentDidMount